mlkem/common.h

```diff
@@ -5,16 +5,36 @@
 #ifndef COMMON_H
 #define COMMON_H
 
-#define DEFAULT_ALIGN 32
+
+/*
+ * C90 does not have the inline compiler directive yet.
+ * We don't use it in C90 builds.
+ * However, in that case the compiler warns about some inline functions in
+ * header files not being used in every compilation unit that includes that
+ * header. To work around it we silence that warning in that case using
+ * __attribute__((unused)).
+ */
+#if !defined(inline)
```

Comment:
Have you tested that this works, since `inline` is a keyword, not a macro (I think)?

We might want to test the C standard here instead?

Reply:
This works in gcc - yes. But you are right this is not a portable solution - it did not work with `msvc` . I re-worked it now to be more portable.